cpp/include/cugraph/graph.hpp

```diff
@@ -483,25 +479,6 @@ class graph_t<vertex_t, edge_t, weight_t, store_transposed, multi_gpu, std::enab
  private:
   friend class cugraph::serializer::serializer_t;
```

Comment:
Could we also get rid of `serializer_t`?  Seems like with `graph_bcast.hpp` gone this class is unused.

Reply:
Done.